dictionaries/CallMap.php

```diff
@@ -9334,8 +9334,8 @@
 'openssl_x509_export_to_file' => ['bool', 'certificate'=>'string|resource', 'output_filename'=>'string', 'no_text='=>'bool'],
 'openssl_x509_fingerprint' => ['string|false', 'certificate'=>'string|resource', 'digest_algo='=>'string', 'binary='=>'bool'],
 'openssl_x509_free' => ['void', 'certificate'=>'resource'],
-'openssl_x509_parse' => ['array|false', 'certificate'=>'string|resource', 'short_names='=>'bool'],
```

Comment:
`short_names` is still optional, so `=` that indicates it has to stay there.

Reply:
Why would `null` be allowed for `short_names`? It results in `TypeError` in PHP 8.

![image](https://user-images.githubusercontent.com/57403/131235509-a09460f3-79f2-4881-9043-f5e83a24aa41.png)

Reply:
Sorry, I was thinking `|null` is the way to indicate it's optional... ;-)